### ISSUE TITLE
Google APIs: Bad version check in static initializers - Error 500

### ISSUE THREAD

**Issue by susanelambrownie** [googleapis/google-http-java-client#1758]
https://issuetracker.google.com/issues/252490306

**Comment by lqiu96** [comment]
Hi @susanelambrownie, thanks for letting us know about this issue. Taking a look to resolve the issues with the static initializers. Would you be able to work with v1.35.2 of `api-client` while we work to resolve this issue?

**Comment by susanelambrownie** [comment]
Hi, after discovering the version check issue, I also discovered 
deprecated methods and classes in the endpoint code.  So I am in the 
process of eliminating the endpoint support from my code which I don't 
really need.  I had also added support for the Android billing client 
which pulls in the newer version of the client lib even if I try to pull 
in an older verson for the endpoints.  And I need to use the newer 
version of the billing client - it will soon be required.   So - good 
luck with the issue but don't worry about me.  Thanks

On 2022-10-25 4:57 pm, Lawrence Qiu wrote:

> Hi @susanelambrownie [1], thanks for letting us know about this issue. 
> Taking a look to resolve the issues with the static initializers. Would 
> you be able to work with v1.35.2 of api-client while we work to resolve 
> this issue?
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>


Links:
------
[1] https://github.com/susanelambrownie
[2] 
https://github.com/googleapis/google-http-java-client/issues/1758#issuecomment-1291133941
[3]
